### PR TITLE
Fix Processing Input not consuming the last Input

### PR DESCRIPTION
**Pull request by KasperOnPC**
Tiny change to fix an issue where the processor was not consuming the last input.

### CODE REVIEW

**Review by thebonbon — APPROVED** [review]

**Review by Arkensor — CHANGES_REQUESTED** [review]
Why does the apple processor take plums now? Maybe that was committed accidently?

**Comment by KasperOnPC** [comment]
> Why does the apple processor take plums now? Maybe that was committed accidentally?

Yeah it was, fixed now, thanks for flagging that.

File: src/Worlds/DebugWorld/DebugWorld_AppleChain.layer [hunk 1]
```diff
@@ -586,5 +586,7 @@ Tree AppleGatherTree : "{1C6F9AD2669C17F9}Prefabs/Vegetation/Tree/t_malus_domest
  angleY -104.782
 }
 GenericEntity AppleProcessor : "{1DD3BF6A1A249CDE}Prefabs/Processor/Apple_Processor.et" {
+ components {
+ }
```
Comment (Arkensor) [inline src/Worlds/DebugWorld/DebugWorld_AppleChain.layer:590]:
```suggestion
```

**Review by Arkensor — CHANGES_REQUESTED** [review]
This can be removed.

File: src/Scripts/Game/Features/Processing/EL_ProcessAction.c [hunk 2]
```diff
@@ -88,22 +66,20 @@ class EL_ProcessAction : ScriptedUserAction
...
-}
+		return true;
+	}
 
+	//------------------------------------------------------------------------------------------------
+	override void Init(IEntity pOwnerEntity, GenericComponent pManagerComponent)
+	{
+		SetCannotPerformReason("Can't find items");
+	}
+}
```
Comment (Arkensor) [inline src/Scripts/Game/Features/Processing/EL_ProcessAction.c:85]:
```suggestion
}

```

**Review by Arkensor — APPROVED** [review]

**Pull request merged**